R/check.R

```diff
@@ -20,31 +22,32 @@
 #'
 #' try(f(x = 1, y = 2, z = 3))
 #' try(f(x = 1, y = 2, 3, 4, 5))
-check_dots_used <- function(env = parent.frame()) {
-  eval_bare(exit_handler, env)
+check_dots_used <- function(env = parent.frame(), action = abort) {
+  eval_bare(exit_handler(action), env)
   invisible()
 }
 
-check_dots <- function(env = parent.frame()) {
+check_dots <- function(env = parent.frame(), action) {
   if (.Call(ellipsis_dots_used, env)) {
     return(invisible())
   }
 
   proms <- dots(env)
   used <- vapply(proms, promise_forced, logical(1))
 
-  unnused <- names(proms)[!used]
```

Comment:
I am _pretty_ sure that `unnused` is just a copy-paste error from `unnamed` and not a weird british spelling of `unused`. Assuming this I fixed the spelling in this PR.